primary/primary.ino

```diff
@@ -58,9 +61,13 @@ void setup() {
 	rotaryEncoder.setup(readEncoderISR);
 	rotaryEncoder.setBoundaries(0, 255 * ROTARY_ENCODER_STEPS, true);
   rotaryEncoder.disableAcceleration();
+
+  raiseDev.begin();
 }
 
 void loop() {
+  raiseDev.updateFirmware("mikey", VERSION_STRING_FROM_GIT);
```

Comment:
☝🏼 is blocking and prevents 👇🏼 from running while it's doing it's little network dance

Reply:
@mikekavouras Oh, good catch. Would this be improved at all if it happened afterwards instead? My guess is "no" but I figured I'd ask anyway.

I'll investigate if this can `yield()` more often during the auto-update process instead. Would it be better if this did anything/checked for updates less frequently/configurably often/not on first run/got everything like NTP setting the times on `setup()` instead?

Appreciate the feedback ❤️